components/test/testSetup.ts

```diff
@@ -10,7 +10,7 @@ import * as EnzymeAdapter from 'enzyme-adapter-react-16'
 // since we are not testing action creators ATM it seemed ok to have it mocked
 // to unblock other work depending on this test refactor.
 // TODO: @cezaraugusto investigate
-jest.mock('../brave_extension/extension/brave_extension/background/actions/shieldsPanelActions.ts')
+jest.mock('../brave_extension/extension/brave_extension/background/actions/shieldsPanelActions')
```

Comment:
turns out this was related to the circular dependency bug. I did `npm run test-unit` and all tests passed without this mock so it's safe to remove

Reply:
Fails for me if I remove this, but passes with it.  So I'll keep it for now and we can figure it out later.